salt/cni/policy/default-kube-dex-policy.yaml.jinja

```diff
@@ -0,0 +1,21 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: kube-dex-policy
+  namespace: kube-system
+spec:
+  podSelector:
+    matchLabels:
+      "app": "dex"
+  egress:
+  - ports:
+# port for LDAP tcp 10.17.1.0:389
+    - port: 389
+      protocol: TCP
+    - port: 6444
+      protocol: TCP
```

Comment:
Humm - we're introducing support for integrating Dex with pre-existing LDAP, SAML or OpenID services. So, Dex will need to ability to connect out to any of those services - which being unmanaged by Kubic/CaaSP, could be on any IP+port combinations.

I wonder how we can handle these scenarios. I suspect we'll need to render specific IPs/ports etc here based on the config data for that. There is probably nothing to do in this PR - just a FYI

Also cc @nanoscopic

Reply:
Yes I would need more input here, what ports dex uses.